media-libs/lib2geom/lib2geom-1.4.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="Easy to use 2D geometry library in C++"
+DESCRIPTION="2Geom is a C++ 2D geometry library geared towards robust processing of computational \
+geometry data associated with vector graphics. The primary design consideration is ease of use \
+and clarity. It is dual-licensed under LGPL 2.1 and MPL 1.1.
```

Comment:
Isn't the license displayed in HaikuDepot anyway? Then having this in the description is redundant.